fusor-ember-cli/app/templates/openshift/openshift-configuration.hbs

```diff
@@ -58,14 +58,29 @@
   }}
 
   <br />
+
   <p>
     Set subdomain name for a wildcard entry to point to the OpenShift router.
   </p>
+
   {{text-f label="Subdomain"
   value=model.openshift_subdomain_name isRequired=true cssId='openshift_subdomain_name'
   disabled=isStarted validator=subdomainValidator
   preText="*." postText=postTextDomainName}}
 
+  <br />
+
+  <p>
+    Selct OpenShift sample applications
+  </p>
+
+  <div class="ose-sample-apps">
+    {{check-f label='Hello World'
+      checked=model.openshift_sample_helloworld
+      isRequired=false
+      helpText='A Hello World sample PHP web application'}}
```

Comment:
Nitpick: It actually is just serving an HTML response. The app I showed in my demo was a PHP app I created.